apidoc/Titanium/Android/HttpResponseCache.yml

```diff
@@ -0,0 +1,149 @@
+---
+name: Titanium.Android.HttpResponseCache
+summary: The Titanium binding of an Android HttpResponseCache.
+description: |
+    Caches HTTP and HTTPS responses to the filesystem so they may be reused, saving time and bandwidth.
+
+    See also [android.net.http.HttpResponseCache](https://developer.android.com/reference/android/net/http/HttpResponseCache.html)
+    in the Android API Reference.
+
+    Cache will be saved in external storage if it present on the moment then
+    [install](Titanium.Android.HttpResponseCache.install) method is called.
+
+    #### Usage example
+
+            if(Ti.Platform.osname == "android") {
+                Ti.Android.HttpResponseCache.setHttpCachePath('_http_');
+                Ti.Android.HttpResponseCache.setHttpCacheSize(30 * 1024 * 1024);  // 30MB
+                Ti.Android.HttpResponseCache.install();
+            }
+
+extends: Titanium.Proxy
+platforms: [android]
+createable: false
+since: "7.1.0"
```

Comment:
Sorry, we clearly missed this release. This will need to be updated for whatever release we can get this in. master is currently targeting 7.4.0, but I think feature freeze may be happening for that real soon.

Reply:
Changed to `7.4.0`